stock_picking_type_bypass_reservation/readme/CONTEXT.md

```diff
@@ -0,0 +1,7 @@
+This module was developed because in certain flows, you need to ship goods before you have registered your stock in Odoo.
+
+It will be useful for you if you want to prepare Pickings or Waves before you have the product in stock or you ship the product before you add stock on the system.
```

Comment:
```suggestion
It will be useful for you if you want to prepare Pickings or Waves before you have the product in stock (and you have partial availability or nothing) or you ship the product before you add stock on the system.
```